.github/workflows/refapp-2x-legacy.yml

```diff
@@ -0,0 +1,38 @@
+name: RefApp 2.x Legacy Selenium Tests
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  repository_dispatch:
+    types: [qa]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      firefox:
+        image: selenium/node-firefox:3.141.59-gold
+        env:
+          HUB_HOST: hub
+          HUB_PORT: 4444
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Checkout qaframework
+        uses: actions/checkout@master
+        with:
+          repository: ${{github.repository}}
+      - name: run db and web containers
+        run: |
+          docker-compose -f docker/docker-compose-refqa.yml up -d
+      - name: wait for openmrs instance to start
+        run: while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8080/openmrs/login.htm)" != "200" ]]; do sleep 1; done
+      - name: Run qaframework on selenium
+        run: |
+          mvn clean install -DskipTests=true
+          cp -f src/test/resources/org/openmrs/uitestframework/test-local.properties src/test/resources/org/openmrs/uitestframework/test.properties
+          cd src/legacy/java
```

Comment:
This is amazing effort @kdaud , One more slight question, does is this folder checked out during the build instantiation ?

Reply:
With ci, the folder is checkout by that command line `cd src/legacy/java`. But locally one has to do it manually!

Reply:
BDD tests does not trigger the selenium legacy tests since their workflow does not check out that specific folder.

Reply:
> 

Is that workflow neccessary since it doesnt trigger anything, what happens if its not there

Reply:
> 

Is that workflow necessary since it doesn't trigger anything, what happens if its not there